wgpu/Cargo.toml

```diff
@@ -64,6 +64,9 @@ glsl = ["naga/glsl-in"]
 ## Enable accepting WGSL shaders as input.
```

Comment:
while we're here we should probably make a note how this interacts with `webgpu` feature: unless `wgsl` is enabled only the `webgpu` backend can accept wgsl

Reply:
Actually there's no special behavior - if the webgl feature isn't available, you can't construct a wgsl variant at all.

Reply:
but if you enable `webgpu` and `webgl` then you can pass in wgsl but only to the webgpu backend unless you also enable the `wgsl` feature, correct?

Reply:
No, to able to use wgsl at all, you need the cfg:

https://github.com/gfx-rs/wgpu/blob/trunk/wgpu%2Fsrc%2Flib.rs#L669-L670

Reply:
ah nice, consistency ✨